src/tls.c

```diff
@@ -3091,14 +3091,14 @@ static int TLSX_CSR_Parse(WOLFSSL* ssl, byte* input, word16 length,
             case WOLFSSL_CSR_OCSP: {
 
                 /* skip responder_id_list */
-                if (length - offset < OPAQUE16_LEN)
+                if ((signed int)(length - offset) < OPAQUE16_LEN)
```

Comment:
The `signed` is implied. For consistency in our code I think this should just be "int".

Reply:
@dgarske, Good point. Done